html/onboarding.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://cdn.tailwindcss.com"></script>
+    <style>
+        :root {
+            --lightbg: #FDE8E9;
+            --lightpri: #E3BAC6;
+            --lightsec: #BC9EC1;
+            --darkpri: #2B59C3;
+            --darksec: #253C78;
+        }
+        @font-face {
+            font-family: overpass;
+            src: url('./src/overpass.ttf');
+        }
+        * {
+            font-family: 'overpass';
+        }
+    </style>
+
+    <title>Craned - On Boarding</title>
+</head>
+<body style="background-color: var(--lightbg);" class="flex flex-col items-center justify-center h-screen gap-10">
+    <div class="flex flex-col items-center p-4 rounded-lg ml-auto mr-auto w-[60%] bg-[var(--lightsec)]">
+        <h1 class="text-2xl text-[var(--darkpri)] font-bold">Profile Customization</h1>
+        <div class="text-lg text-white text-left w-full indent-5">To get started with Craned, please give us some information. Note that your existing information will be automatically filled, but feel free to change it if you'd like.</div>
+        <form class="w-[80%]">
+            <div class="flex justify-center w-full gap-1">
+                <div class="flex flex-col gap-1 bg-[var(--lightpri)] p-3 rounded-lg items-center w-4/5">
+                    <input type="file" id="pfpupload" class="hidden">
+                    <label for="pfpupload" class="bg-[var(--darkpri)] rounded-full p-2 w-[70px] h-[70px] flex items-center">
+                        <svg xmlns="http://www.w3.org/2000/svg" class="h-[25px] w-[50px]" viewBox="0 0 384 512">
+                            <!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. -->
+                            <style>svg{fill:#ffffff}</style>
+                            <path d="M96 128a128 128 0 1 1 256 0A128 128 0 1 1 96 128zM0 482.3C0 383.8 79.8 304 178.3 304h91.4C368.2 304 448 383.8 448 482.3c0 16.4-13.3 29.7-29.7 29.7H29.7C13.3 512 0 498.7 0 482.3zM504 312V248H440c-13.3 0-24-10.7-24-24s10.7-24 24-24h64V136c0-13.3 10.7-24 24-24s24 10.7 24 24v64h64c13.3 0 24 10.7 24 24s-10.7 24-24 24H552v64c0 13.3-10.7 24-24 24s-24-10.7-24-24z"/>
+                        </svg>
+                    </label>
```

Comment:
So, the upload button is basically the profile picture? So does that mean if the profile picture exist, we use that image instead, but if user didn't have the profile images at first, it will be default color?